x/bank/msgs.go

```diff
@@ -27,6 +28,49 @@ func (msg MsgSend) Type() string  { return "send" }
 
 // Implements Msg.
 func (msg MsgSend) ValidateBasic() sdk.Error {
+	if msg.FromAddress.Empty() {
+		return sdk.ErrInvalidAddress("No FromAddress included")
+	}
+	if msg.ToAddress.Empty() {
+		return sdk.ErrInvalidAddress("No ToAddress included")
+	}
+	if !msg.Amount.IsPositive() {
+		return sdk.ErrInsufficientCoins("Send Amount must be positive")
+	}
+	return nil
```

Comment:
Should we also add validation checks to the amount integer size a la #1008?

Reply:
How much do we bound it by?

Reply:
Good question, thoughts @cwgoes?